pkg/http/server.go

```diff
@@ -267,6 +273,35 @@ func GenerateHashMetrics(cfg config.MutualPeersConfig) error {
 	return nil
 }
 
+// ConsNodesIDs generates the metric with the consensus nodes ids.
+func ConsNodesIDs(cfg config.MutualPeersConfig) error {
+	log.Info("Generating the metric for the consensus nodes ids...")
+
+	for _, mutualPeer := range cfg.MutualPeers {
```

Comment:
it's better to have an iteration logged in errors, so we can understand more verbosely what is going on